pyro/burgers/problems/smooth.py

```diff
@@ -0,0 +1,42 @@
+import sys
+
+import numpy
+
+import pyro.mesh.patch as patch
+from pyro.util import msg
+
+
+def init_data(my_data, rp):
+    """ initialize the smooth burgers problem """
+
+    msg.bold("initializing the smooth burgers problem...")
+
+    # make sure that we are passed a valid patch object
+    if not isinstance(my_data, patch.CellCenterData2d):
+        print("ERROR: patch invalid in smooth.py")
+        print(my_data.__class__)
+        sys.exit()
+
+    u = my_data.get_var("x-velocity")
+    v = my_data.get_var("y-velocity")
+
+    xmin = my_data.grid.xmin
+    xmax = my_data.grid.xmax
+
+    ymin = my_data.grid.ymin
+    ymax = my_data.grid.ymax
+
+    xctr = 0.5*(xmin + xmax)
+    yctr = 0.5*(ymin + ymax)
+
+    A = 0.8
```

Comment:
What is A here? Maybe define the problem at the top of the file, see e.g. incompressible/problems/shear.py